netbox/netbox/configuration.example.py

```diff
@@ -261,7 +261,7 @@
 # https://docs.djangoproject.com/en/stable/ref/templates/builtins/#date
 DATE_FORMAT = 'N j, Y'
 SHORT_DATE_FORMAT = 'Y-m-d'
-TIME_FORMAT = 'g:i a'
+TIME_FORMAT = 'H:i:s'
```

Comment:
Why have you changed the default formats?

Reply:
I separated this change into it’s own commit to describe my intend:

> a.m./p.m. is illogical. Military time format can be pronounced as "sixteen hundred and fifty-two".

But that is only the format that appears on hover now. Should I drop that change or the whole commit? Are you fine with `SHORT_DATETIME_FORMAT = 'Y-m-d H:i:s'`?

Reply:
Let's just leave the defaults in the example alone. If there's a case to change them, it should be addressed under a separate issue.

Reply:
I dropped the commit.